ui/src/components/DownloadButton/DownloadButton.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactElement } from 'react';
+import Button from '@splunk/react-ui/Button';
+
+interface DownloadButtonProps {
+    // needs to be same domain if not it will just open link
+    fileUrl: string;
+    fileNameAfterDownload: string;
+    children: ReactElement | string;
+}
+
+function DownloadButton(props: DownloadButtonProps) {
+    return (
+        <a
+            target="_blank"
+            href={props.fileUrl}
+            rel="noopener noreferrer"
+            download={props.fileNameAfterDownload}
+            data-test="downloadButton"
+        >
+            <Button>{props.children}</Button>
```

Comment:
SUI team recommends to use Link element. Buttons should be used for actions only, Links for URLs (even it is a URLs to file).
https://splunk.slack.com/archives/C8R00E97S/p1647392758667479